packages/wdio-browserstack-service/src/insights-handler.ts

```diff
@@ -528,7 +812,7 @@ class _InsightsHandler {
             }
         }
 
-        if (eventType === 'TestRunStarted') {
+        if (eventType === 'TestRunStarted' || eventType === 'TestRunSkipped') {
```

Comment:
In the context of cucumber a test is skipped when the step in feature file doesn't match with any of the step definitions. Do we still get the same `TestRunSkipped` event ? If not then we may need to handle that scenario

Reply:
That case will get handled in TestRunFinished event with result as skipped. This `TestRunSkipped` event is a custom event sent for cases when a before hook fails